sdk/ai/azure-ai-agents/azure/ai/agents/aio/operations/_patch.py

```diff
@@ -2081,9 +2081,9 @@ async def get_last_message_by_role(
         :return: The most recent message authored by *role* in the thread, or None if no such message exists.
         :rtype: Optional[~azure.ai.agents.models.ThreadMessage]
         """
-        pageable = await self.list(thread_id, **kwargs)  # type: ignore[arg-type]
+        pageable = self.list(thread_id, **kwargs)  # type: ignore[arg-type]
 
-        for message in pageable:
+        async for message in pageable:
```

Comment:
This should be in the [jhakulin/list-op-update](https://github.com/Azure/azure-sdk-for-python/tree/jhakulin/list-op-update), did you got latests merged?

Reply:
Fixed